src/librustc/mir/interpret/mod.rs

```diff
@@ -47,7 +47,7 @@ pub struct GlobalId<'tcx> {
     /// For a promoted global, the `Instance` of the function they belong to.
     pub instance: ty::Instance<'tcx>,
 
-    /// The index for promoted globals within their function's `Mir`.
+    /// The index for promoted globals within their function's `Body`.
```

Comment:
The doc comments should probably say `mir::Body` just to be more clear (let me know if you want me to find all occurrences and comment on them).

Reply:
I'll try to find them. I'll make my changes on a separate commit so it's easier for you to find.